pkg/controller/direct/compute/firewallpolicyrule/mapper.generated.go

```diff
@@ -0,0 +1,138 @@
+// Copyright 2024 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package firewallpolicyrule
+
+import (
+	pb "cloud.google.com/go/compute/apiv1/computepb"
+	krm "github.com/GoogleCloudPlatform/k8s-config-connector/apis/compute/v1beta1"
+	"github.com/GoogleCloudPlatform/k8s-config-connector/pkg/controller/direct"
+)
+
+func ComputeFirewallPolicyRuleSpec_FromProto(mapCtx *direct.MapContext, in *pb.FirewallPolicyRule) *krm.ComputeFirewallPolicyRuleSpec {
+	if in == nil {
+		return nil
+	}
+	out := &krm.ComputeFirewallPolicyRuleSpec{}
+	out.Action = in.GetAction()
+	out.Description = in.Description
+	out.Direction = in.GetDirection()
+	out.Disabled = in.Disabled
+	out.EnableLogging = in.EnableLogging
+	out.Match = FirewallpolicyruleMatch_FromProto(mapCtx, in.Match)
+	out.Priority = int64(in.GetPriority())
+	// MISSING: RuleName
```

Comment:
For the missing fields, can we add a comment as to why? Something like "not yet or not supported in proto for now, etc.)

Reply:
Right, those fields we are not supported yet. Instead of adding comment here in the generated code, I feel we may want to mention that `"MISSING" unsupported fields for Beta resources is accepted` in the [guide](https://github.com/GoogleCloudPlatform/k8s-config-connector/blob/b5cc2e924037a053af625b1781aed2a71b7caf67/docs/develop-resources/guides/3-add-mapper.md#exit-criteria), WDYT?